chart/values.yaml.tmpl

```diff
@@ -56,3 +56,4 @@ isAirgap: false
 userAgent: ""
 replicatedID: ""
 appID: ""
+additionalMetricsEndpoint: ""
```

Comment:
thoughts on supporting multiple endpoints here? asking because it's what copilot kept trying to guide me towards :)

Reply:
that's actually a good idea.